src/domains/checkout-app/04_apim_checkout_payment_wallet.tf

```
@@ -0,0 +1,73 @@
+locals {
+  apim_checkout_payment_wallet_api = {
+    display_name          = "Checkout pagoPA Payment Wallet"
+    description           = "Checkout APIs dedicated to wallet domain integration."
+    path                  = "checkout/payment-wallet"
+    subscription_required = true
```

```suggestion
    subscription_required = false
```

those are api to be exposed to frontend (checkout) so should not be protected by api key